docs/Inleiding.md

```diff
@@ -0,0 +1,43 @@
+# Inleiding
```

Comment:
Wellicht in dit verhaal nog een aantal andere voordelen benoemen die met deze aanpak in zicht komen zoals 
* generatie van documentatie vanuit de Informatiemodellen;
* verschillenrapportage tussen versies van Informatiemodellen;
* versnelde vervaardiging van de specificaties;
* ...

Reply:
Toegevoegd